### PR TITLE
Replace MomentJs with Dayjs

### PR DESCRIPTION
**Pull request by hotzenklotz**
In preparation to upgrade `antd` to v5, they recommend to switch from `MomentJs` to `Dayjs`. While both have an almost identical API, `dayjs` is said to be a lot smaller in file size and features immutable operations. 

For the most part I could replace any `moment()` call with `dayjs()` and it just works.

Antd Docs on this topic: https://4x.ant.design/docs/react/replace-moment

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Run type checker
- Other tests are unclear... many views use time formatting and durations.

### Issues:
- None

------
(Please delete unneeded items, merge only when none are left open)
- [x] Updated [changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes
- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change
- [ ] Removed dev-only changes like prints and application.conf edits
- [ ] Considered [common edge cases](../blob/master/.github/common_edge_cases.md)
- [ ] Needs datastore update after deployment

### CODE REVIEW

File: frontend/javascripts/admin/statistic/statistic_view.tsx [hunk 1]
```diff
@@ -52,17 +57,11 @@ class StatisticView extends React.PureComponent<{}, State> {
   };
 
   componentDidMount() {
-    moment.updateLocale("en", {
-      week: {
-        dow: 1,
```
Comment (hotzenklotz) [inline frontend/javascripts/admin/statistic/statistic_view.tsx:57]: Moved to dayjs "setup" in `format_utils`

File: frontend/javascripts/libs/format_utils.ts [hunk 2]
```diff
@@ -164,24 +193,25 @@ export function formatSeconds(durationSeconds: number): string {
...
-  const duration = moment.duration(durationInMillisecons);
-  const minuteDuration = duration.minutes() + 60 * duration.hours();
-  const minutesAsString = `${minuteDuration < 10 ? 0 : ""}${minuteDuration}`;
-  const hoursAsSeconds = `${duration.seconds() < 10 ? 0 : ""}${duration.seconds()}`;
-  return `${minutesAsString}:${hoursAsSeconds}`;
+  const duration = dayjs.duration(durationInMillisecons);
+  // const minuteDuration = duration.minutes() + 60 * duration.hours();
+  // const minutesAsString = `${minuteDuration < 10 ? 0 : ""}${minuteDuration}`;
+  // const hoursAsSeconds = `${duration.seconds() < 10 ? 0 : ""}${duration.seconds()}`;
+  // return `${minutesAsString}:${hoursAsSeconds}`;
```
Comment (philippotto) [inline frontend/javascripts/libs/format_utils.ts:200]: remove?

**Review by philippotto — APPROVED** [review]
great :+1: just see my one comment before merging.

File: frontend/javascripts/libs/format_utils.ts (hunk 2 above)
Comment (hotzenklotz) [reply frontend/javascripts/libs/format_utils.ts]: Done

**Pull request merged**